### PR TITLE
Bug 2002291: [4.7]: userspace: Don't sync every iptables rule on every…

### PR DESCRIPTION
**Pull request by danwinship**
backport of #930

### CODE REVIEW

File: pkg/proxy/userspace/proxier.go [hunk 1]
```diff
@@ -377,8 +379,9 @@ func (proxier *Proxier) syncProxyRules() {
 		return
 	}
 
-	if err := iptablesInit(proxier.iptables); err != nil {
-		klog.Errorf("Failed to ensure iptables: %v", err)
+	if proxier.forceReload {
+		if err := iptablesInit(proxier.iptables); err != nil {
+			klog.Errorf("Failed to ensure iptables: %v", err)
```
Comment (zhaozhanqi) [inline pkg/proxy/userspace/proxier.go:384]: @danwinship  lack one `}` here

Comment (danwinship) [reply pkg/proxy/userspace/proxier.go:384]: oops, fixed

**Pull request merged**